### DISCUSSION TITLE
ScriptReference/Windows.CrashReporting-crashReportFolder

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#582] [Notes]
# ScriptReference/Windows.CrashReporting-crashReportFolder

https://docs.unity3d.com/ScriptReference/Windows.CrashReporting-crashReportFolder.html

**Comment by litefeel** [comment]
在Windows10上实际路径是 %USERPROFILE%/AppData/Local/temp/(CompanyName)/(ProjectName)/Crashes
